autopilot/contractor/contractor.go

```diff
@@ -1059,33 +911,39 @@ func performContractFormations(ctx *mCtx, bus Bus, cr contractReviser, ipFilter
 	// select hosts, since we already have all of them in memory we select
 	// len(candidates)
 	candidates = candidates.randSelectByScore(len(candidates))
-	if len(candidates) < wanted {
+	if uint64(len(candidates)) < wanted {
 		logger.Warn("not enough candidates to form new contracts")
 	}
 
 	// get the initial contract funds
 	minInitialContractFunds := InitialContractFunding
 
 	// form contracts until the new set has the desired size
+	var nFormed uint64
 	for _, candidate := range candidates {
 		if wanted == 0 {
-			return formedContracts, nil // done
+			return nFormed, nil // done
 		}
 
 		// break if the autopilot is stopped
 		select {
 		case <-ctx.Done():
-			return nil, context.Cause(ctx)
+			return 0, context.Cause(ctx)
 		default:
 		}
 
 		// prepare a logger
 		logger := logger.With("hostKey", candidate.host.PublicKey).
 			With("addresses", candidate.host.ResolvedAddresses)
 
+		// create separate usability logger
+		usability := logger.Named("usability").
```

Comment:
This is a bit overkill. A new contract is always good. No need to log that information.